libraries.md

```diff
@@ -1,9 +1,6 @@
 # Third-Party Libraries
 
-- jQuery: Available throughout HQ. We use jQuery 2 ([2.2.0](https://github.com/dimagi/commcare-hq/blob/master/bower.json)) but are in the process of migrating to jQuery 3.0, which has a number of breaking changes. What does this mean for you?
-  - Check out the [migration guide](https://jquery.com/upgrade-guide/3.0), or just skim the [commcare-hq migration branch's commits](https://github.com/dimagi/commcare-hq/compare/jquery-3-1-0) to see the patterns most commonly used in HQ that need to be updated. 
-  - Avoid constructs that are will break in 3+
-  - For now, also avoid constructs that are only available in 3+
+- jQuery: Available throughout HQ. We use jQuery 3 ([3.2.1](https://github.com/dimagi/commcare-hq/blob/master/bower.json)) everywhere except in Web Apps, which depends on [marionette](https://marionettejs.com/) v2, which is only compatible with jQuery 2.
```

Comment:
Maybe don't include the specific version of jquery here since it's likely to change before this doc is updated. Suggested edit:
```
... We use [jQuery 3](https://github.com/dimagi/commcare-hq/blob/master/bower.json) everywhere except ...
```

Reply:
good point, will update